src/viam/sdk/components/private/board_client.cpp

```diff
@@ -137,6 +139,33 @@ Board::digital_value BoardClient::read_digital_interrupt(const std::string& digi
     return response.value();
 }
 
+void BoardClient::stream_ticks(const std::vector<std::string> digital_interrupt_names,
+                               std::shared_ptr<std::queue<tick>> ticks,
+                               const AttributeMap& extra) {
+    viam::component::board::v1::StreamTicksRequest request;
```

Comment:
Could this implementation use the `make_client_helper` used elsewhere in this file? It is OK if it can't for some reason (though I'd like to know why), since it is a helper and not intended to cover 100% of cases, but I'm always interested to know if there are cases that it currently can't cover but could be made to.

Reply:
I don't think it works, the line `make_client_helper(this, *stub_, &StubType::StreamTicks)` has error  `no instance of function template "viam::sdk::make_client_helper" matches the argument list`. Maybe because its a streaming API?

Reply:
Quite likely. Would you mind filing a ticket to investigate why make_client_helper doesn't work with streaming? It should be made to, if possible.

Reply:
https://viam.atlassian.net/browse/RSDK-7318